services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java

```diff
@@ -978,8 +978,11 @@ public void modifyMeterForFlow(DatapathId dpid, long meterId, long bandwidth)
     }
 
     @Override
-    public Map<DatapathId, IOFSwitch> getAllSwitchMap() {
-        return ofSwitchService.getAllSwitchMap();
+    public Map<DatapathId, IOFSwitch> getAllSwitchMap(boolean visible) {
```

Comment:
Do we need anywhere "not ready to be used switches"? I mean when we need to set/use "false" for visible argument?

Reply:
underlying fl logic uses it however doesn't expose, I decided to provide the flag as well.
Use case: don't know, might be something in future